lib/services/matchers.go

```diff
@@ -369,6 +374,14 @@ var SupportedAWSMatchers = []string{
 	AWSMatcherMemoryDB,
 }
 
+// RequireAWSIAMRolesAsUsersMatchers is a list of the AWS databases that
+// require AWS IAM roles as database users.
+// TODO(gavin): if we add database matchers for AWS keyspaces, OpenSearch, or
+// DynamoDB discovery, add them here.
```

Comment:
Should we convert this TODO to a comment, like "IMPORTANT: if you add ..."?

Are there other places where this warning could be relevant?

Is this a likely place for folks adding new matchers to look?

Reply:
I'll change it to IMPORTANT: ... 

I think it's likely that it will be seen, but I can add a comment in api/types/database.go where we have a check for the same thing

Reply:
Thanks.